src/chttpd_db.erl

```diff
@@ -93,7 +93,7 @@ handle_changes_req1(#httpd{}=Req, Db) ->
             couch_stats:decrement_counter([couchdb, httpd, clients_requesting_changes])
         end;
     _ ->
-        Msg = <<"Supported `feed` types: normal, continuous, longpoll, eventsource">>,
+        Msg = <<"Supported `feed` types: normal, continuous/stream, longpoll, eventsource">>,
```

Comment:
a bit of clarification, I most care about the error message being clear and I suggest we use @kxepal’s version, if we add the alias. I don’t advocate for or against the alias, but if others think, that it is a good idea, you should go ahead.